pkg/cli/admin/release/new.go

```diff
@@ -1497,26 +1494,6 @@ func hasTag(tags []imageapi.TagReference, tag string) *imageapi.TagReference {
 	return nil
 }
 
-func pruneEmptyDirectories(dir string) error {
```

Comment:
Isn't this function used by anywhere anymore?. Do you happen to know why?.

Reply:
I just grepped `git grep pruneEmptyDirectories` but yeah, let me grep the git history to get more context.

/hold

For context why I'm digging into this code, I'm looking into [OCPBUGS-19824](https://issues.redhat.com/browse/OCPBUGS-19824) and [OCPBUGS-30080](https://issues.redhat.com/browse/OCPBUGS-30080). I'll have more PRs like this.

Reply:
Haha, brought in by Clayton in 2018 (62d9e2c6358291547b9282a9e8f37f8f6681e4d4)... unused :rofl: 

```console
$ git log -S pruneEmptyDirectories
commit 773d900edd34c706efee594eda94d45bebab6cb1
Author: Petr Muller <muller@redhat.com>
Date:   Thu Nov 14 13:05:36 2024 +0100

    adm release: remove unused code

commit 62d9e2c6358291547b9282a9e8f37f8f6681e4d4
Author: Clayton Coleman <ccoleman@redhat.com>
Date:   Tue Jul 31 13:46:50 2018 -0400

    Add a prototype release command
    ...

$ git checkout 62d9e2c6358291547b9282a9e8f37f8f6681e4d4 --quiet
$ git grep pruneEmptyDirectories
pkg/oc/cli/admin/release/new.go:func pruneEmptyDirectories(dir string) error {
```

/hold cancel